charts/agent/templates/_helpers.tpl

```diff
@@ -333,26 +376,33 @@ secure.enabled=false we need to explicitly put those config entires in the
 agent config to prevent a backend push from enabling them after installation.
 */}}
 {{- define "agent.secureFeatures" }}
-    {{- $secureBlockConfig := dict "security" (dict
-        "enabled" .Values.secure.enabled
-        "k8s_audit_server_enabled" .Values.auditLog.enabled) }}
+    {{- if .Values.auditLog.enabled }}
+        {{- if or (and (hasKey .Values.global.sysdig "secure")
+                       (not .Values.global.sysdig.secure))
+                   .Values.secureLight }}
+            {{- $_ := set .Values.auditLog "enabled" false }}
```

Comment:
Is there a way to have a (debug?) message to the user to notify them that we're overruling what they asked?

Reply:
I'm not sure this should be in the scope of this PR